static/docs/user-guide/contributing-documentation.md

```diff
@@ -12,10 +12,10 @@ To contribute documentation you need to know these locations:
 - [Content](https://github.com/iterative/dvc.org/tree/master/static/docs)
   (`/static/docs`) -
   [Markdown](https://guides.github.com/features/mastering-markdown/) files of
-  the different pages to render dynamically in the browser;
+  the different pages to render dynamically in the browser
```

Comment:
This bullet should be a sentence so the list is consistent (the other 2 bullets are sentences and end in period `.`). Let's also switch to `:` instead of `-` here. Please use:

```
- [Content](https://github.com/iterative/dvc.org/tree/master/static/docs)
  (`/static/docs`): Place
  [Markdown](https://guides.github.com/features/mastering-markdown/) files of
  the different pages to render dynamically in the browser.
```